src/display/svg.js

```diff
@@ -832,8 +849,7 @@ SVGGraphics = (function SVGGraphicsClosure() {
       current.txtElement.setAttributeNS(XML_NS, 'xml:space', 'preserve');
       current.txtElement.appendChild(current.tspan);
       current.txtgrp.appendChild(current.txtElement);
-
-      this._ensureTransformGroup().appendChild(current.txtElement);
+      this._clipAndAppendToGroup(current.txtgrp);
```

Comment:
`txtgrp` should be `txtElement` I think, since that was in the old code, or is that wrong?